<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicSkinSettings</class>
 <widget class="QWidget" name="ClassicSkinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&amp;Skin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="orientation_label">
       <property name="text">
        <string>&amp;orientation</string>
       </property>
       <property name="buddy">
        <cstring>orientation_cbox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orientation_cbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="spacing_label">
       <property name="text">
        <string>spaci&amp;ng</string>
       </property>
       <property name="buddy">
        <cstring>spacing_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spacing_edit">
       <property name="minimum">
        <number>-8</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="ignore_advance_y">
     <property name="text">
      <string>ignore predefined &amp;vertical distance between characters</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>348</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="background_group">
     <property name="title">
      <string>&amp;background</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="bg_per_element_cb">
        <property name="text">
         <string>apply per character</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="bg_solid_color_rbtn">
        <property name="text">
         <string>solid color</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="bg_pattern_rbtn">
        <property name="text">
         <string>pattern</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="bg_select_pattern_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="bg_gradient_rbtn">
        <property name="text">
         <string>gradient</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="bg_select_color_btn">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="bg_select_gradient_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="bg_pattern_stretch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>stretch instead of tile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="texture_group">
     <property name="title">
      <string>te&amp;xture</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="tx_solid_color_rbtn">
        <property name="text">
         <string>solid color</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="tx_pattern_rbtn">
        <property name="text">
         <string>pattern</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="tx_select_gradient_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tx_select_color_btn">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="tx_per_element_cb">
        <property name="text">
         <string>apply per character</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="tx_select_pattern_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="tx_gradient_rbtn">
        <property name="text">
         <string>gradient</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="tx_pattern_stretch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>stretch instead of tile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="ignore_advance_x">
     <property name="text">
      <string>ignore predefined &amp;horizontal distance between characters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tx_solid_color_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>tx_select_color_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tx_gradient_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>tx_select_gradient_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tx_pattern_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>tx_select_pattern_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_solid_color_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>bg_select_color_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_gradient_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>bg_select_gradient_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_pattern_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>bg_select_pattern_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tx_pattern_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>tx_pattern_stretch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_pattern_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>bg_pattern_stretch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
